kafka-event-listener/src/main/java/org/hypertrace/core/kafka/event/listener/KafkaLiveEventListener.java

```diff
@@ -54,13 +63,15 @@ public void close() throws Exception {
   }
 
   public static final class Builder<K, V> {
-    List<BiConsumer<? super K, ? super V>> callbacks = new ArrayList<>();
+    Queue<BiConsumer<? super K, ? super V>> deprecatedCallbacksFlow = new ConcurrentLinkedQueue<>();
```

Comment:
nit - looks like these were these exposed to package before? Since we're technically breaking it here anyway, does it need to be?

naming - this isn't really a flow, it's used by the flow. I'd just leave it as callbacks and mark it deprecated